src/modelgauge/tests/safe_v1.py

```diff
@@ -172,34 +182,28 @@ class SafeTestVersion1(BaseSafeTestVersion1):
 ]
 
 
-def create_safe_test_v1_uid(hazard, locale, postfix=None):
-    if postfix is None:
-        postfix = ""
-    return f"safe-{hazard}-{locale.value}-1.0{postfix}".lower()
-
-
-def register_tests(cls, postfix=None):
+def register_tests(cls, evaluator=None):
     for locale in Locale:
         for hazard in cls.hazards:
-            test_uid = create_safe_test_v1_uid(hazard, locale, postfix=postfix)
-            TESTS.register(cls, test_uid, hazard, locale, ALL_PERSONAS)
+            test_uid = BaseSafeTestVersion1.create_uid(hazard, locale, evaluator)
+            if not test_uid in TESTS.keys():
```

Comment:
I'm curious about the reasoning behind adding this check. The test registry raises an error when you attempt to register multiple tests to the same UID. This is the preferred behavior IMHO, because otherwise you don't know which test is actually being used if execution continues normally.

Reply:
Ah, got it. Something was blowing up, so it seemed to me that making this idempotent was reasonable. Could you say more about the case you're concerned about? I

Reply:
I found what blows up. In `get_benchmark` it calls `ensure_ensemble_annotators_loaded` if it needs an ensemble annotator. And If we're doing multiple languages, that gets called multiple times. So I'd like something to be idempotent, but it doesn't have to be this bit.